simple/stats/config.md

```diff
@@ -132,6 +186,28 @@ Use "/" as a separator to specify a multi-level hierarchy.
 An array of NL sentences to be used for creating more NL embeddings (in addition to the description)
 for the variable.
 
+## `entities`
```

Comment:
I lean towards not having this field. If needed, user should just add name and description columns in the csv?

Reply:
Can you elaborate - not sure how they can be in the CSV?

Also, this is optional. If they don't provide one, the name will be same as the type.